xp0-double/xp0d-azuredeploy-msdeploy.json

```diff
@@ -116,18 +116,6 @@
       "minLength": 8,
       "defaultValue": "[concat(toUpper(uniqueString('cd-web')), '@', uniqueString(parameters('sqlserver_password')))]"
     },
-    "analytics_mongodb_connectionstring": {
-      "type": "securestring"
-    },
```

Comment:
your absolutely right, my mistake - I used the provisioning output for this, so my edit made these parameters redundant; can indeed be removed